js/index.js

```diff
@@ -1,7 +1,64 @@
+var canvas = document.querySelector('canvas')
+var ctx = canvas.getContext('2d')
+let intervalId
 window.onload = () => {
   document.getElementById('start-button').onclick = () => {
-    startGame();
-  };
+    startGame()
+  }
+}
+const road = new Image()
+road.src = './images/road.png'
 
-  function startGame() {}
-};
+const backgroundImage = {
+  img: road,
+  y: 0,
+  speed: -1,
+
+  draw: function() {
+    this.y++
+    if (this.y < -canvas.width) {
```

Comment:
Su problema con la imagen está dentro de esta pieza de código.
Lo que quieren es que las imagenes bajen (y--) entonces lo que les preoocupa es cuando la imagen sale del canvas por la parte inferior (el limite de canvas o lo que mide de alto).
```suggestion
    if (this.y > canvas.width) {
```